### PR TITLE
getting started: changed the intro and the first steps section

### PR DESCRIPTION
**Pull request by jonasbru**

### CODE REVIEW

File: rest_api/v2/info/index.yml [hunk 1]
```diff
@@ -3,21 +3,55 @@ version: "v2"
...
+    - Retrieve data on businesses you have access to if you are a publisher.
     - ...
 
   # JS SDK
+  To know how to integrate Partoo in a white label mode using our JS SDK, please check our [JS SDK documentation](/js_sdk/).
+
+  # Getting started
+
+  ## Security Disclaimer
+  **The Partoo Rest API is meant to be used only from a server that you control.** The API Key should **never** be transmitted
```
Comment (lachaib) [inline rest_api/v2/info/index.yml:21]: Here the reader doesn't know yet what an API Key is. I would start the paragraph with something like:
Authentication to Rest API is based on API Keys which are attached to users (human or bot).

or a link to the description of API Keys

File: rest_api/v2/info/index.yml [hunk 2]
```diff
@@ -3,21 +3,55 @@ version: "v2"
...
 
-  To know how to integrate Partoo views using our JS SDK, please check our [JS SDK documentation](/js_sdk/).
+  Ask your Partoo Account Manager to send you an API Key for Sandbox. You'll need to pass this API Key in all your API calls in order
+  to be authentified, in the header `x-APIKey`. More details in the <a href="#section/Authentication">Authentication section</a>.
 
-  # Resources structure
+  From there, you can call any route that your user has access to. To know the exact API endpoint to use, you can click
+  on the route url above the "Response samples", and this will give you the urls for Production and Sandbox.
+
+  If you require an UI access to Sandbox, ask your Partoo Account Manager to give you one. Beware that Sandbox environment will be empty by default,
```
Comment (theobk) [inline rest_api/v2/info/index.yml:44]:
```suggestion
  If you require a UI access to Sandbox, ask your Partoo Account Manager to give you one. Beware that Sandbox environment will be empty by default,
```
https://www.a-or-an.com/a_an/ui

File: rest_api/v2/info/index.yml [hunk 3]
```diff
@@ -3,21 +3,55 @@ version: "v2"
...
 
   # JS SDK
+  To know how to integrate Partoo in a white label mode using our JS SDK, please check our [JS SDK documentation](/js_sdk/).
+
+  # Getting started
+
+  ## Security Disclaimer
+  **The Partoo Rest API is meant to be used only from a server that you control.** The API Key should **never** be transmitted
+  to the front-end, and should be kept secret.
+  If you require to make calls from a front-end, you should proxy them by a server that you control, that will add the API Key.
```
Comment (camille-dev) [inline rest_api/v2/info/index.yml:23]:
```suggestion
  If you need to make calls from a front-end, you should proxy them by a server that you control, that will add the API Key.
```

File: rest_api/v2/info/index.yml [hunk 4]
```diff
@@ -3,21 +3,55 @@ version: "v2"
...
+
+  ## Environments
+  We provide two public environments, **Production** and **Sandbox**.
+  - **Production** is where your final data should be.
```
Comment (camille-dev) [inline rest_api/v2/info/index.yml:27]: not sure I understand the sentence "where your final data should be". Actual/Real data?

File: rest_api/v2/info/index.yml [hunk 5]
```diff
@@ -3,21 +3,55 @@ version: "v2"
...
+  - **Sandbox** is a playground environment for you to test your API integration without having to worry about side effects.
+
+  Some functionalities are disabled in Sandbox, to avoid having a production impact:
+  - Business data will not be sent to the different publishers as GMB, Facebook, etc.
```
Comment (camille-dev) [inline rest_api/v2/info/index.yml:31]:
```suggestion
  - Business data will not be sent to the different publishers such as GMB, Facebook, etc.
```

File: rest_api/v2/info/index.yml (hunk 2 above)
Comment (camille-dev) [inline rest_api/v2/info/index.yml:44]:
```suggestion
  If you need an UI access to Sandbox, ask your Partoo Account Manager for one. Beware that Sandbox environment will be empty by default,
```

File: rest_api/v2/info/index.yml (hunk 4 above)
Comment (JeanneL-Partoo) [reply rest_api/v2/info/index.yml:27]: Maybe by changing and presenting sandbox first then production, the difference between will be more understandable ?

File: rest_api/v2/info/index.yml (hunk 2 above)
Comment (JeanneL-Partoo) [inline rest_api/v2/info/index.yml:44]:
```suggestion
  If you require an UI access to Sandbox, ask your Partoo Account Manager to give you one. Be aware that Sandbox environment will be empty by default,
```

```suggestion
  If you require an UI access to Sandbox, ask your Partoo Account Manager to give you one. Beware, Sandbox environment is empty by default,
```

File: rest_api/v2/info/index.yml (hunk 1 above)
Comment (jonasbru) [reply rest_api/v2/info/index.yml]: I've rewritten the paragraph. Thanks !

File: rest_api/v2/info/index.yml (hunk 4 above)
Comment (jonasbru) [reply rest_api/v2/info/index.yml]: I've set sandbox first, and rewritten the prod description. Thanks !

**Pull request merged**